### PR TITLE
Help edits for Sprint 95

### PR DESCRIPTION
**Pull request by marcfishelquantil**
These edits cover the changes in Sprint 95.

### CODE REVIEW

File: docs/portal/managing-standby-tasks.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Managing Standby Tasks
```
Comment (burtonquantil) [inline docs/portal/managing-standby-tasks.md:1]: The use of "tasks" isn't quite consistent with our portal.  The user can have a standby task for staging and one for production at the same time. The user can add one or more deployment and undeployment actions to each task. There isn't technically a standby task for each certificate and property undeployment and deployment as implied by your phrasing.  The portal's interface represents what is supported by our API.  https://docs.cdnetworks.com/cdn/apidocs#operation/createDeployment (The call can specify multiple actions to do in one deployment task.)

**Review by burtonquantil — CHANGES_REQUESTED** [review]
Marc, please check my comments.

File: docs/portal/certificates/updating-certificates.md [hunk 2]
```diff
@@ -9,31 +9,31 @@ CDN360 sends an email notification when a certificate used in production is clos
...
 <p align="center"><img src="/docs/resources/images/certificates/certificate-versions-upload.png" alt="Upload Certificate Version" width="700"></p>
-5. Scroll down to the <strong>Deployment</strong> section, and then select <strong>Production</strong> from the <strong>Deployment Destination</strong> drop-down list to deploy the new version to production.<br><br>
+6. Scroll down to the <strong>Deployment</strong> section, and then select <strong>Production</strong> from the <strong>Deployment Destination</strong> drop-down list to deploy the new version to production.<br><br>
 <strong>Note:</strong> To view the deployment history of the property before deploying it, click the <strong>Deployment History</strong> button.</br></br>
-6. Click <strong>Deploy Configuration</strong>.</br><br>
-
+7. To deploy the certificate now, click <strong>Deploy Now</strong>. To deploy it at a later time, click <strong>Add to Standby task</strong> to add this task to the [standby queue](</docs/portal/managing-standby-tasks.md>).
```
[the remark this answers is not in the document]
Comment (marcfishelquantil) [reply docs/portal/certificates/updating-certificates.md:20]: Yes, it needed to be updated. I fixed this, but the reason I did not do so initially was that I believe we no longer use this figure in the help. Now I realize that if we no longer use a figure, I will simply delete it. Again, thank you.

File: docs/portal/managing-standby-tasks.md (hunk 1 above)
Comment (marcfishelquantil) [reply docs/portal/managing-standby-tasks.md:1]: Got it, thank you. Please see updated wording in my next upload.

**Review by burtonquantil — APPROVED** [review]
I was debating internally whether using "subqueue" is good. There are technically two different deployment tasks that can be created, one for production and one for staging.

**Pull request merged**